docs/governance/politeia.md

```diff
@@ -2,54 +2,56 @@
 
 ---
 
-**What is Politeia?**
+## What is Politeia?
 
-Politeia, or Pi, is a platform which is being developed to support Decred’s governance. It facilitates the submission, tracking, and discussion of Decred governance proposals by the stakeholder community.
+Politeia is a platform that supports Decred’s governance. It facilitates the submission, tracking, and discussion of Decred governance proposals by the stakeholder community. The Politeia [web platform](https://proposals.decred.org/) facilitates browsing, discussing and submitting proposals.
 
-The Politeia web platform (at [proposals.decred.org](https://proposals.decred.org/)) facilitates browsing, discussing and submitting proposals.
+Reddit-style **up/down voting** is used for comment sorting. Up/down votes are not anonymous.
 
-**Politeia proposals**
+Voting on proposals does not happen directly on Politeia, but within official Decred wallets ([Decrediton](../getting-started/user-guides/decrediton-setup.md) or [politeiavoter CLI](https://github.com/decred/politeia/tree/master/politeiavoter) with dcrwallet).
```

Comment:
This might be better phrased as something along the lines of:

Voting on proposals does not happen directly on Politeia because it requires signatures from your Decred wallet.  Decrediton has a built in voting tool or you can use the politeiavoter CLI.

Reply:
Reworded to:

> Voting on proposals does not happen directly on Politeia, as it requires signatures from your Decred wallet. Voting can be done from [Decrediton](../getting-started/user-guides/decrediton-setup.md) or on the command line using the [politeiavoter CLI](https://github.com/decred/politeia/tree/master/politeiavoter) tool.